src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -679,7 +746,9 @@ void TableDockWidget::prepareDataForPolly(QString writableTempDir,
 }
 
 void TableDockWidget::exportJsonToPolly(QString writableTempDir,
-                                        QString jsonfileName) {
+                                        QString jsonfileName,
+                                        bool uploadToPolly)
```

Comment:
`uploadToPolly` is misleading, no? this should be `uploadtoPollyPhi` if we're only sending for fluxomics workflow, or `true` every time because it's `exportJsonToPolly`. @rish9511

Reply:
> this should be uploadtoPollyPhi if we're only sending for fluxomics workflow or true every time because it's exportJsonToPolly.

I think you meant  `pollyPhi` workflow. It can't be true every time since we also upload data to fluxomics. 

> uploadToPolly is misleading, no? 

Yes, to some extent. If we are uploading to pollyPhi  i.e if `uploadToPolly` is `true` then we add some extra content(data labelled by ML model) to json files. Maybe we can rename it to `addExtraData` ?

Reply:
Sorry if I wasn't clear. I meant that if we are only sending the extra data for PollyPhi workflow, the variable can be renamed to `uploadtoPollyPhi` and set true when PollyPhi is selected. Or we can keep it as `uploadToPolly` and add the extra data in all Polly JSONs.

I'm in favour of changing the variable name to either `uploadToPollyPhi` or `addMLInfo`. What do you think?

Reply:
>  Or we can keep it as uploadToPolly and add the extra data in all Polly JSONs

They don't want this data in FirstView

> I'm in favour of changing the variable name to either uploadToPollyPhi or addMLInfo. What do you think?

addMLInfo sounds better